x/stakingplus/spec/01_state.md

```diff
@@ -0,0 +1,7 @@
+<!--
+order: 1
+-->
+
+# State
+
+There is no difference in state with that of Staking module of the Cosmos-SDK. Refer to the [original document](../../staking/spec/01_state.md) for more information.
```

Comment:
How about using `LBM-SDK` instead of `Cosmos-SDK`?

Reply:
Do we have to change all the Cosmos things in the specs? I think it's also a good option to describe only the differences between the original documents. Then, the original documents could be referenced by the permanent links, including the version.

If we are planning to change the x/staking spec files in #488, it's plausible to apply your suggestion. However, it appears to be quite opposite to the existence of this module, because all the modifications should be in x/stakingplus, so there would be no differences between x/staking and that of Cosmos-SDK. Then, the documents on x/staking would remain intact. OR, we could write documents on x/staking, including the statements saying there is no difference between the module and the upstream's, and the reference to the original Cosmos-SDK's document. In this case, changing the name to LBM-SDK should be made. But, why should we?

Reply:
OK, I understood. I think it's better not fix it. Thank you.